dev-requirements.txt

```diff
@@ -1,3 +1,7 @@
 django-debug-toolbar==1.5     # https://django-debug-toolbar.readthedocs.io/en/stable/changes.html
 django-extensions==1.7.4
+ipython==5.1.0
+ipython-genutils==0.1.0
+ipdb==0.10.1
+notebook==4.2.3
```

Comment:
See the updated PR description above, and the (identical) amended commit message.